addons/stock/tests/test_old_rules.py

```diff
@@ -602,9 +614,9 @@ def test_pick_ship_1(self):
         self.assertEqual(picking_pick.origin, move1.group_id.name)
 
         # second out move, the "pick" picking should have lost its partner and have its origin updated
-        move2._action_confirm()
+        moves._action_confirm()
         self.assertEqual(picking_pick.partner_id.id, False)
-        self.assertEqual(picking_pick.origin, f'{move1.group_id.name},{move2.group_id.name}')
+        self.assertEqual(picking_pick.origin, f'{move1.group_id.name},{moves.group_id.name}')
```

Comment:
with the above suggestion you will need to change this line, I would suggest to assign 
```py
move2, move3 = self.env['stock.move'].create([{
```
Instead of moves = ... and then keep the assert as it was:
```suggestion
        self.assertEqual(picking_pick.origin, f'{move1.group_id.name},{move2.group_id.name}')
```